docs/developer.qmd

```diff
@@ -1,4 +1,311 @@
-# Developer guide: Create your own Tab
-## Synopsis
-## Code structure
-## How to
+# Developer guide
+
+## Submitting issues
+If you want to report a bug please make sure to mention:
```

Comment:
It would also be helpful to know the python version and the irods version of the server.

Reply:
Yes, I tried it. But we do not have GitHub markdown pages but GitHub.io pages with quarto. I will try to figure something out. But otherwise this is for the next update.